packages/coding-standard/tests/Fixer/Naming/StandardizeHereNowDocKeywordFixer/StandardizeHereNowDocKeywordFixerTest.php

```diff
@@ -20,9 +20,16 @@ public function test(SmartFileInfo $fileInfo): void
         $this->doTestFileInfo($fileInfo);
     }
 
-    public function provideData(): Iterator
+    public function provideData(): AppendIterator
     {
-        return StaticFixtureFinder::yieldDirectory(__DIR__ . '/Fixture');
+        $appendIterator = new AppendIterator();
+        $appendIterator->append(StaticFixtureFinder::yieldDirectory(__DIR__ . '/Fixture'));
+
+        if (PHP_VERSION_ID >= 70300) {
+            $appendIterator->append(StaticFixtureFinder::yieldDirectory(__DIR__ . '/Fixture73plus'));
+        }
+
+        return $appendIterator;
```

Comment:
This should be split into 2 test files instead.
One with `@requires`

This is good test case to mimic: https://github.com/symplify/symplify/blob/fcc595d6d463f3758315a7f8383df913b1b0a736/packages/coding-standard/tests/Fixer/Commenting/ParamReturnAndVarTagMalformsFixer/Php74Test.php#L14

Reply:
implemented.